libkbfs/keybase_service_base.go

```diff
@@ -681,3 +684,19 @@ func (k *KeybaseServiceBase) GetPublicCanonicalTLFNameAndID(
 
 	return res, nil
 }
+
+// EstablishMountDir asks the service for the current mount path
+func (k *KeybaseServiceBase) EstablishMountDir(ctx context.Context) (
+	string, error) {
+	dir, err := k.kbfsMountClient.GetCurrentMountDir(ctx)
+	if err != nil || dir == "" {
+		dirs, _ := k.kbfsMountClient.GetAllAvailableMountDirs(ctx)
+		// chooseDefaultMount can handle an emtpy argument
+		dir, err = chooseDefaultMount(dirs)
```

Comment:
It's strange to ignore the error here, but pass it along to the caller (who fails when they get the error). Is the overall mount operation supposed to succeed when `dirs` is empty or is it supposed to fail?  If the former, you'll need to fix up the caller of this function.  If the later, you should return immediately when you hit an error.